docs/usermanual.md

```diff
@@ -23,15 +23,16 @@ by the device to the agent. Passive or lazy attributes, i.e. those attributes wh
 request from the agent, are not implemented. Please check the issue
 [#89](https://github.com/telefonicaid/iotagent-json/issues/89) for more details and updates regarding its implementation.
 
-### <a name="httpbinding"/> HTTP binding
+###  HTTP binding
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated